### PR TITLE
Add READMEquay.md

### PR DESCRIPTION
**Pull request by pecameron**
This describes using quay.io to automatically build images from your local github/<user>/ovn-kubernetes master branch. The images can be "docker pulled" into nodes in the cluster and be used as your development image.
    
Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@rajatchopra @dcbw @squeed PTAL

**Comment by pecameron** [comment]
@rajatchopra @dcbw PTAL

File: README.md [hunk 1]
```diff
@@ -243,3 +243,4 @@ does install kubernetes in its simplest form.
 [INSTALL.SSL.md]: docs/INSTALL.SSL.md
 [config.md]: docs/config.md
 [HA.md]: docs/ha.md
+
```
Comment (rajatchopra) [inline README.md]: @pecameron  Did you still want this commit and the change?

File: READMEquay.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Building with quay.io
+
+## quay.io repositories
+
+quay.io is a convenient place to build and store you development
```
Comment (dcbw) [inline READMEquay.md]: you -> your

File: READMEquay.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Dockerfile.centos builds the binaries and then includes them in the 
+image.
+
+
+## setup docker pull access on each host in the cluster
+Edit **/etc/containers/registries.conf**
+
+```
+registries:
+  - brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888
```
Comment (dcbw) [inline READMEquay.md]: I'd take this registry out since it's an internal one that nobody else can use.

File: READMEquay.md (hunk 2 above)
Comment (pecameron) [reply READMEquay.md]: Fixed

File: READMEquay.md (hunk 3 above)
Comment (pecameron) [reply READMEquay.md]: Fixed

**Comment by pecameron** [comment]
@rajatchopra @dcbw @squeed PTAL

File: READMEquay.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+```
+quay.io/repository/<your-account>/ovn-kube:latest
+```
+You can use the image in your daemonsets for development.
+
+Dockerfile.centos builds the binaries and then includes them in the 
+image.
+
+
+## setup docker pull access on each host in the cluster
```
Comment (squeed) [inline READMEquay.md]: I'm not sure this is necessary if you use a full URL (e.g. quay.io/foo/bar)

File: README.md (hunk 1 above)
Comment (pecameron) [reply README.md]: @rajatchopra the changes to README.md are out of scope. README.md needs a significant revision.

File: READMEquay.md (hunk 4 above)
Comment (pecameron) [reply READMEquay.md]: @squeed I deleted the section

**Pull request merged**